src/storage/src/hummock/state_store.rs

```diff
@@ -296,7 +297,12 @@ impl StateStore for HummockStorage {
 
     define_state_store_associated_type!();
 
-    fn get<'a>(&'a self, key: &'a [u8], epoch: u64) -> Self::GetFuture<'_> {
+    fn get<'a>(
+        &'a self,
+        key: &'a [u8],
+        epoch: u64,
+        _vnode: Option<VirtualNode>,
```

Comment:
Considering there might be not such a "PointGet" operator, I think the type of vnodes should also be `Vec<VirtualNode>`. Nevermind, it's not a big problem.

Reply:
I'd prefer always using `&'a VirtualNode`, so that it will function efficiently even when vnode mapping is large.

Reply:
> I'd prefer always using &'a VirtualNode, so that it will function efficiently even when vnode mapping is large.

Will `Option<&'a VirtualNode>` still cause some overhead due to the construction of `Option`?

Reply:
Nope, it has exactly the same value size as `&'a VirtualNode`.